test/circuit_breaker_test.rb

```diff
@@ -403,19 +403,45 @@ def test_success_count_on_error_rate_calculation
       assert notifier.notified?, 'no notification sent'
     end
 
+    def test_send_execution_time_metric
+      notifier = gimme_notifier(metric: :execution_time, metric_value: Gimme::Matchers::Anything.new)
+      circuit = Circuitbox::CircuitBreaker.new(:yammer, notifier_class: notifier)
+      circuit.run { 'success' }
+      assert notifier.metric_sent?, 'no execution time metric sent'
+    end
+
+    def test_no_execution_time_metric_on_error_execution
+      notifier = gimme_notifier(metric: :execution_time, metric_value: Gimme::Matchers::Anything.new)
+      circuit = Circuitbox::CircuitBreaker.new(:yammer, notifier_class: notifier)
+      circuit.run { raise Timeout::Error }
+      refute notifier.metric_sent?, 'execution time metric sent'
+    end
+
+    def test_no_execution_time_metric_when_circuit_open
+      notifier = gimme_notifier(metric: :execution_time, metric_value: Gimme::Matchers::Anything.new)
+      circuit = Circuitbox::CircuitBreaker.new(:yammer, notifier_class: notifier)
+      10.times { circuit.run { raise Timeout::Error }}
```

Comment:
Can we do this without the need to run the circuit 10 times, perhaps stubbing ```open?```. Then we would be able to test that when the circuit is open this doesn't run. It looks like this test is actually testing that we don't see any metrics sent when we see failures and then when the circuit is open, assuming it actually opened.

I see this is done in other tests in this file and I think we should change those separately.